src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -529,3 +534,13 @@ fun Policy.getDisallowedActions(allowList: List<String>): List<String> {
  * Allowed actions are ones that are specified in the [ManagedIndexSettings.ALLOW_LIST] setting.
  */
 fun Action.isAllowed(allowList: List<String>): Boolean = allowList.contains(this.type.type)
+
+fun isMetadataMoved(
```

Comment:
return true meaning metadata is moved. Ideally, there won't be any node update cluster state metadata, so it won't be more recently updated than config index metadata.

Failed to clean cluster state metadata, it should be safe to ignore it. Since if it's moved, it won't be moved again.

Still added a warning log